best_practices/forms.rst

```diff
@@ -159,34 +159,54 @@ remove the ``form_widget(form)`` function and render your fields individually.
 See :doc:`/form/form_customization` for more information on this and how you
 can control *how* the form renders at a global level using form theming.
 
+As the whole rendering of a form can be passed to a form theme, it can be
+usefull to do something like that:
+
+.. code-block:: html+twig
+
+    <!-- templates/random.html.twig -->
+    {% form_theme someform 'templates/form/themes/_someform.html.twig' %}
+
+    {{ form(someform) }}
+
+This way, the entire form rendering is passed to the theme where you can
+control all field independently.
+
 Handling Form Submits
 ---------------------
 
-Handling a form submit usually follows a similar template::
+Handling the form submission can be a tricky part of your application but
+as soon as you understand how it works, your code become suddently way easier to maintain.
+In order to ease the process, we recommend to use FormHandler for every form, this way,
```

Comment:
I am okay with adding a new "form handler" chapter. But I am not convinced that we should recommend it as a best practice.